recipe/meta.yaml

```diff
@@ -40,23 +40,27 @@ requirements:
     - exprtk
     - gtest
     - libarrow =15
+    - libboost-headers
     - libparquet =15
     - libprotobuf
     - librdkafka
     - lz4-c
     - numpy
+    - openssl
     - pip
     - pyarrow =15
     - rapidjson
     - ruamel.yaml
     - scikit-build
     - setuptools >=69
+    - websocketpp
     - zlib
   run:
     - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
     - python
     - {{ pin_compatible('numpy') }}
     - backports.zoneinfo  # [py<39]
+    - openssl
```

Comment:
You don't need to repeat this in `run`. `run_exports` takes care of this.

Reply:
Ah ok was following the docs, will delete next pr